config/rbac/role.yaml

```diff
@@ -2,7 +2,6 @@
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
-  creationTimestamp: null
```

Comment:
shouldn't this have also labels?
    app.kubernetes.io/name: build-service
    app.kubernetes.io/managed-by: kustomize